### PR TITLE
Added new function to get pvc creation & deletion time for pvc_list

### PR DESCRIPTION
**Pull request by ramkiperiy**
Added new function to get pvc creation & deletion time for pvc_list
#988

Signed-off-by: rperiyas <rperiyas@redhat.com>

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -949,6 +949,86 @@ def measure_pvc_creation_time(interface, pvc_name):
...
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_csi_provisioner_pod(interface)
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(pod_name[0], 'csi-provisioner')
+    logs += pod.get_pod_logs(pod_name[1], 'csi-provisioner')
+
+    pvc_dict = dict()
+    format = '%H:%M:%S.%f'
+    for pvc_obj in pvc_obj_list:
+        logs = logs.split("\n")
```
Comment (keesturam) [inline tests/helpers.py]: can you move this outside for loop? We don't want to split the static log file every time.

File: tests/helpers.py [hunk 2]
```diff
@@ -949,6 +949,86 @@ def measure_pvc_creation_time(interface, pvc_name):
     return total.total_seconds()
 
 
+def measure_pvc_creation_time_bulk(interface, pvc_obj_list):
+    """
+    Measure PVC creation time of bulk PVC based on logs.
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_obj_list (list): List of PVC Names for measuring creation time
```
Comment (keesturam) [inline tests/helpers.py]: `pvc_obj_list` might be misleading as they are not objects and just names. can you change the name of this parameter?

Comment (ramkiperiy) [reply tests/helpers.py]: It's pvc_obj_list only, if we expect pvc_name_list then while passing need to get name of each pvc object and store in list. I will change the args description, in next commit.

File: tests/helpers.py (hunk 1 above)
Comment (ramkiperiy) [reply tests/helpers.py]: ack

File: tests/helpers.py (hunk 2 above)
Comment (ramkiperiy) [reply tests/helpers.py]: changed it back to pvc_name_list, resolving comment.

**Comment by ramkiperiy** [comment]
Local run logs for measuring creation and deletion of all 4 types of PVCs http://pastebin.test.redhat.com/829431
Local run logs for measuring creation and deletion time of 500 PVCs is in attachment 
[logs.txt](https://github.com/red-hat-storage/ocs-ci/files/4095966/logs.txt)

**Pull request merged**